components/event-publisher-proxy/pkg/env/nats_config.go

```diff
@@ -0,0 +1,26 @@
+package env
+
+import "time"
+
+// Config represents the environment config for the Event Publisher NATS.
+type NatsConfig struct {
+	Port                int           `envconfig:"INGRESS_PORT" default:"8080"`
+	NatsPublishURL      string        `envconfig:"NATS_URL" default:"nats.nats.svc.cluster.local"`
+	MaxIdleConns        int           `envconfig:"MAX_IDLE_CONNS" default:"100"` // ??
```

Comment:
Do we need the `// ??` at the end of the line?

Reply:
Right, removed cause it's not used for NATS.